Tests/ClientTests/Coordinators/BrowserCoordinatorTests.swift

```diff
@@ -378,6 +378,7 @@ final class BrowserCoordinatorTests: XCTestCase {
     // MARK: - Settings route
 
     func testGeneralSettingsRoute_showsGeneralSettingsPage() throws {
+        guard !CoordinatorFlagManager.isSettingsCoordinatorEnabled else { return }
```

Comment:
Can you inject this value instead so it can be mocked or hard coded in the tests?